src/App.js

```diff
@@ -148,7 +162,12 @@ export default function App() {
 
     function doMove(moveEvent) {
```

Comment:
I think this one might also be ok to move into the Menu component. the only things it needs access to are the pokemon object which is already been passed into that component, disable menu (which you could pass in as a prop) and returnToMain which could be also moved to Menu since it really is mostly used in there already